auction-server/src/auction.rs

```diff
@@ -1348,8 +1378,19 @@ impl ChainStore for ChainStoreSvm {
     }
 
     async fn get_trigger_stream<'a>(client: &'a Self::WsClient) -> Result<Self::TriggerStream<'a>> {
-        let (slot_subscribe, _) = client.slot_subscribe().await?;
-        Ok(slot_subscribe)
+        let (block_subscribe, _) = client
+            .block_subscribe(
+                solana_client::rpc_config::RpcBlockSubscribeFilter::All,
+                Some(RpcBlockSubscribeConfig {
+                    encoding:                          None,
+                    transaction_details:               Some(TransactionDetails::None),
+                    show_rewards:                      Some(false),
+                    max_supported_transaction_version: None,
+                    commitment:                        Some(CommitmentConfig::confirmed()),
```

Comment:
is it possible to listen for `finalized` blocks?
we want to sign the transaction with this block hash and it's safer to use `finalized` for this